CHANGELOG.md

```
@@ -30,4 +30,8 @@
         - Provides a more intuitive mental model for developers when dealing with potential failures.
     - **Migration:**
         - Existing code utilizing the `Result` class will need to update their generic type parameters accordingly (e.g., `Result<int, String>` becomes `Result<String, int>`).
-        - No functionality is changed other than type parameter order, so the code should work the same way after the fix, providing the types are updated correctly.
\ No newline at end of file
+        - No functionality is changed other than type parameter order, so the code should work the same way after the fix, providing the types are updated correctly.
+
+ ## 1.2.1
```

Markdownlint (0.37.0)

Headings must start at the beginning of the line

(MD023, heading-start-left)